examples/app.R

```diff
@@ -288,9 +288,20 @@ sidebar <- function() {
           div(class="menu",
               a(class="item", href="#accordion", "Accordion"),
               a(class="item", href="#rating", "Rating"),
-              a(class="item", href="#tabset", "Tabset")
+              a(class="item", href="#tabset", "Tabset"),
+              a(class="item", href="#calendar", "Calendar")
               )))
 }
+
+calendar <- function() {
+  div(
+    h1(class="ui header", id="calendar", "Calendar"),
+    demo(uicalendar("date", type = "date", value = "20.2.2020", placeholder = "Select Date", min = "2.2.2020", max = "25.2.2020")),
```

Comment:
did that pass linter? can you break the line?

Reply:
My mistake, I didn't run it. It had length 132 so wouldn't pass.
Fixed it + fixed small style issues in calendar.R